sonoff.server.module.js

```diff
@@ -228,9 +242,15 @@ module.exports.createServer = function (config) {
                                 device.messages = device.messages.filter(function (item) {
                                     return item !== message;
                                 })
-                                device.state = message.params.switch;
-                                state.updateKnownDevice(device);
-                                log.trace('INFO | WS | APP | action has been accnowlaged by the device ' + JSON.stringify(data));
+
+                                device.isAlive = true;
+                                
+                                if(message.params.switches){
```

Comment:
could you please add an "else" where the old behaviour is triggered? so that the other devices will still work?